int32 inputMotor
---
string outResponse

